frontend/src/reducers/story.js

```diff
@@ -0,0 +1,46 @@
+import { HANDLE_SICK, HANDLE_TESTED, LOADING, SAVED_STORY, SAVE_STORY_START } from '../actions/types';
+
+const initialState = {
+    status: {},
+    sick: null,
+    tested: null,
```

Comment:
As I was saying above, we coul remove these two, and on `SAVED_STORY` save the tested & sick status. Then, where these are used, access `story`.

Reply:
I agree with you, but perhaps is better doing this improvement in another PR so this ons stops growing